OpenRA.Mods.Common/UtilityCommands/CheckSequenceSprites.cs

```diff
@@ -23,12 +23,14 @@ public bool ValidateArguments(string[] args)
 			return true;
 		}
 
+		int exit;
```

Comment:
This can be a local `var failed = false;`, and then choose the exit code at the very end.